### PR TITLE
Updated duplicate package names

### PR DESCRIPTION
**Pull request by Rutzzzz**
Change of package names so that they are unique across all QS.

**Check list**:

Your pull request:

- [x] targets the `development` branch
- [x] uses the `999-SNAPSHOT` version of Quarkus
- [ ] has tests (`mvn clean test`)
- [ ] works in native (`mvn clean package -Pnative`)
- [ ] has native tests (`mvn clean verify -Pnative`)
- [ ] makes sure the documentation must not be updated
- [ ] links the documentation update pull request (if needed)
- [ ] updates or creates the `README.md` file (with build and run instructions)
- [ ] For new quickstart, is located in the directory _component-quickstart_
- [ ] For new quickstart, is added to the root `pom.xml` and `README.md`

### CODE REVIEW

**Comment by Rutzzzz** [comment]
I tried to make the package names unique. I did not necessairly follow any convention, although lot of packages do follow the naming pattern 'org.acme.QUICKSTART_NAME'.

I agree that having a unified approach would be best. 

org.acme.QUICKSTART_KEYWORD seems to be the most logical choice for the reasons you have pointed out.

File: getting-started-async/src/main/java/org/acme/quickstart/async/GreetingResource.java [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package org.acme.quickstart;
+package org.acme.quickstart.async;
```
Comment (gsmet) [inline getting-started-async/src/main/java/org/acme/quickstart/async/GreetingResource.java]: Some of them have `quickstart` in it but not all of them? Should we make everything consistent? @rsvoboda WDYT?

BTW, we will need to upgrade the guides too.

Comment (rsvoboda) [reply getting-started-async/src/main/java/org/acme/quickstart/async/GreetingResource.java]: +1 to have it consistent

I asked @Rutzzzz to prepare list of proposed package names.

He should paste the list here so we can agree on it before he starts adjusting the code.

**Comment by Rutzzzz** [comment]
On second thought I propose the convention 'org.acme.QUICKSTART_NAME'. 

The adventage is unambiguity in translation from quickstart names. Just few names are long and I think it is acceptable (or perhaps we could cut those a little bit shorter). 

The 'org.acme.QUICKSTART_KEYWORD' convention might sometimes be difficult to pull off without conflicts and with clear purpose of the quickstart.

The proposed package names:
./amqp-quickstart
    org.acme.amqp
./cache-quickstart
    org.acme.cache
./config-quickstart
    org.acme.config
./dynamodb-quickstart
    org.acme.dynamodb
./getting-started-async
    org.acme.getting.started.async
./getting-started-knative
    org.acme.getting.started.knative
./getting-started
    org.acme.getting.started
./getting-started-testing
    org.acme.getting.started.testing
./hibernate-orm-panache-quickstart
    org.acme.hibernate.orm.panache
./hibernate-orm-quickstart
    org.acme.hibernate.orm
./hibernate-search-elasticsearch-quickstart
    org.acme.hibernate.search.elasticsearch
./infinispan-client-quickstart
    org.acme.infinispan.client
./jms-quickstart
    org.acme.jms
./kafka-quickstart
    org.acme.kafka
./kafka-streams-quickstart/aggregator
    org.acme.kafka.streams.aggregator
./kafka-streams-quickstart/producer
    org.acme.kafka.streams.producer
./kogito-quickstart
    org.acme.kogito
./lifecycle-quickstart
    org.acme.lifecycle
./microprofile-fault-tolerance-quickstart
    org.acme.microprofile.fault.tolerance
./microprofile-health-quickstart
    org.acme.microprofile.health
./microprofile-metrics-quickstart
    org.acme.microprofile.metrics
./mongodb-panache-quickstart
    org.acme.mongodb.panache
./mongodb-quickstart
    org.acme.mongodb
./mqtt-quickstart
    org.acme.mqtt
./neo4j-quickstart
    org.acme.neo4j
./openapi-swaggerui-quickstart
    org.acme.openapi.swaggerui
./opentracing-quickstart
    org.acme.opentracing
./quartz-quickstart
    org.acme.quartz
./qute-quickstart
    org.acme.qute
./reactive-routes-quickstart
    org.acme.reactive.routes
./rest-client-multipart-quickstart
    org.acme.rest.client.multipart
./rest-client-quickstart
    org.acme.rest.client
./rest-json-quickstart
    org.acme.rest.json
./scheduler-quickstart
    org.acme.scheduler
./security-jdbc-quickstart
    org.acme.security.jdbc
./security-jwt-quickstart
    org.acme.security.jwt
./security-keycloak-authorization-quickstart
    org.acme.security.keycloak.authorization
./security-oauth2-quickstart
    org.acme.security.oauth2
./security-openid-connect-quickstart
    org.acme.security.openid.connect
./security-openid-connect-web-authentication-quickstart
    org.acme.security.openid.connect.web.authentication
./software-transactional-memory-quickstart
    org.acme.software.transactional.memory
./spring-data-jpa-quickstart
    org.acme.spring.data.jpa
./spring-di-quickstart
    org.acme.spring.di
./spring-security-quickstart
    org.acme.spring.security
./spring-web-quickstart
    org.acme.spring.web
./tests-with-coverage-quickstart
    org.acme.tests.with.coverage
./tika-quickstart
    org.acme.tika
./validation-quickstart
    org.acme.validation
./vertx-quickstart
    org.acme.vertx
./websockets-quickstart
    org.acme.websockets

**Comment by rsvoboda** [comment]
@Rutzzzz here is your proposal put in nicer way, just 2 global string replaces of your list and a header. The form matters ;)

@gsmet are you ok with proposed package names ?

| QS        | Package name |          
| ------------- |:-------------|
| amqp-quickstart| org.acme.amqp
| cache-quickstart| org.acme.cache
| config-quickstart| org.acme.config
| dynamodb-quickstart| org.acme.dynamodb
| getting-started-async| org.acme.getting.started.async
| getting-started-knative| org.acme.getting.started.knative
| getting-started| org.acme.getting.started
| getting-started-testing| org.acme.getting.started.testing
| hibernate-orm-panache-quickstart| org.acme.hibernate.orm.panache
| hibernate-orm-quickstart| org.acme.hibernate.orm
| hibernate-search-elasticsearch-quickstart| org.acme.hibernate.search.elasticsearch
| infinispan-client-quickstart| org.acme.infinispan.client
| jms-quickstart| org.acme.jms
| kafka-quickstart| org.acme.kafka
| kafka-streams-quickstart/aggregator| org.acme.kafka.streams.aggregator
| kafka-streams-quickstart/producer| org.acme.kafka.streams.producer
| kogito-quickstart| org.acme.kogito
| lifecycle-quickstart| org.acme.lifecycle
| microprofile-fault-tolerance-quickstart| org.acme.microprofile.fault.tolerance
| microprofile-health-quickstart| org.acme.microprofile.health
| microprofile-metrics-quickstart| org.acme.microprofile.metrics
| mongodb-panache-quickstart| org.acme.mongodb.panache
| mongodb-quickstart| org.acme.mongodb
| mqtt-quickstart| org.acme.mqtt
| neo4j-quickstart| org.acme.neo4j
| openapi-swaggerui-quickstart| org.acme.openapi.swaggerui
| opentracing-quickstart| org.acme.opentracing
| quartz-quickstart| org.acme.quartz
| qute-quickstart| org.acme.qute
| reactive-routes-quickstart| org.acme.reactive.routes
| rest-client-multipart-quickstart| org.acme.rest.client.multipart
| rest-client-quickstart| org.acme.rest.client
| rest-json-quickstart| org.acme.rest.json
| scheduler-quickstart| org.acme.scheduler
| security-jdbc-quickstart| org.acme.security.jdbc
| security-jwt-quickstart| org.acme.security.jwt
| security-keycloak-authorization-quickstart| org.acme.security.keycloak.authorization
| security-oauth2-quickstart| org.acme.security.oauth2
| security-openid-connect-quickstart| org.acme.security.openid.connect
| security-openid-connect-web-authentication-quickstart| org.acme.security.openid.connect.web.authentication
| software-transactional-memory-quickstart| org.acme.software.transactional.memory
| spring-data-jpa-quickstart| org.acme.spring.data.jpa
| spring-di-quickstart| org.acme.spring.di
| spring-security-quickstart| org.acme.spring.security
| spring-web-quickstart| org.acme.spring.web
| tests-with-coverage-quickstart| org.acme.tests.with.coverage
| tika-quickstart| org.acme.tika
| validation-quickstart| org.acme.validation
| vertx-quickstart| org.acme.vertx
| websockets-quickstart| org.acme.websockets

**Comment by gsmet** [comment]
I would make:
 - `org.acme.microprofile.fault.tolerance` -> `org.acme.microprofile.faulttolerance` (as in SmallRye)
 - `org.acme.tests.with.coverage` -> `org.acme.testcoverage`

Otherwise, it looks good.

Note that the guides will also need adjustments.

Thanks!

**Comment by Rutzzzz** [comment]
I have renamed the packages as suggested. If you're okay with the changes, I will go ahead and update the guides.

**Comment by gsmet** [comment]
AFAICS, this needs a rebase. Please ping me when it's done, I'll merge as soon as CI is green.

**Pull request merged**